packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -27,16 +27,18 @@ const double _kLeadingWidth = kToolbarHeight; // So the leading button is square
 
 // Bottom justify the kToolbarHeight child which may overflow the top.
 class _ToolbarContainerLayout extends SingleChildLayoutDelegate {
-  const _ToolbarContainerLayout();
+  const _ToolbarContainerLayout(this.titleHeight) : assert(titleHeight != null);
+
+  final double titleHeight;
```

Comment:
This isn't just the height of the title, it's the height of the toolbar (the NavigationToolBar built by the AppBar). It would be better to call it toolbarHeight. That would also help one understand why the default is kToolbarHeight :-). The docs for toolbarHeight should make its effect on leading/title/actions clear.

Reply:
Okay